### ISSUE TITLE
Website License

### ISSUE THREAD

**Issue by elioqoshi** [opensourcedesign/organization#60]
We currently have no license stated for our website content, which means that we reserve all rights to it.

We should change that, I'd propose a CC-BY-SA 4.0 International license for all our content on our website (apart where otherwise noted).

We should also notify all people posting to job boards that their content will be licensed under the same license. 

What do you think?

**Comment by jancborchardt** [comment]
Yes, absolutely agree. :) All our content should be [Creative Commons Attribution & Share-Alike](https://creativecommons.org/licenses/by-sa/4.0/).

cc @bnvk @simonv3 @evalica @belenbarrospena @victoria-bondarchuk

**Comment by elioqoshi** [comment]
Judging from the likes on the OP I think that we can move forward with this :)
@simonv3 can we implement this?

**Comment by simonv3** [comment]
Don't need my permission!
